### PR TITLE
libbacktrace: add support for compiler=msvc in profile

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **lib/1.0**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by SpaceIm** [comment]
@jgsogo @danimtb No build for gcc11? I've also opened this PR to generate gcc11 binaries since it's required for https://github.com/conan-io/conan-center-index/pull/8712 (actually for boost which is a dependency of libcds).

**Review by SSE4 — APPROVED** [review]

**Review by uilianries — APPROVED** [review]
LGTM

File: recipes/libbacktrace/all/conanfile.py [hunk 1]
```diff
@@ -79,7 +85,8 @@ def _configure_autotools(self):
             return self._autotools
         self._autotools = AutoToolsBuildEnvironment(self, win_bash=tools.os_info.is_windows)
         self._autotools.libs = []
-        if self.settings.compiler == "Visual Studio" and tools.Version(self.settings.compiler.version) >= "12":
+        if (self.settings.compiler == "Visual Studio" and tools.Version(self.settings.compiler.version) >= "12") or \
+           str(self.settings.compiler) == "msvc":
```
Comment (ericLemanissier) [inline recipes/libbacktrace/all/conanfile.py:89]: why is there no version check for msvc ?

Comment (SpaceIm) [reply recipes/libbacktrace/all/conanfile.py:89]: I don't trust stability of msvc versionning in settings.yml currently. Moreover, it begins at Visual Studio 2015.

**Review by ericLemanissier — APPROVED** [review]

**Pull request merged**